File: Uri-Online-Judge/Wellington/C:C++/Iniciante/1038.cpp
#include <stdio.h>

int main(){
    double lanche[5] = {4.00, 4.50, 5.00, 2.00, 1.50};
    int codigo, quantidade;
    double total;
    
    scanf("%i", &codigo);
    scanf("%i", &quantidade);
    
    total = quantidade * lanche[codigo-1];
    
    printf("Total: R$ %.2lf\n", total);
    
    return 0;
}

File: Uri-Online-Judge/Wellington/Python 3/Iniciante/1038.py
# -*- coding: utf-8 -*-

linha = list(map(int, input().split()))
codigo, quantidade = linha

if codigo == 1:
	print('Total: R$ {:.2f}'.format(quantidade * 4))
elif codigo == 2:
	print('Total: R$ {:.2f}'.format(quantidade * 4.5))
elif codigo == 3:
	print('Total: R$ {:.2f}'.format(quantidade * 5))
elif codigo == 4:
	print('Total: R$ {:.2f}'.format(quantidade * 2))
elif codigo == 5:
	print('Total: R$ {:.2f}'.format(quantidade * 1.5))